### ISSUE TITLE
A11y_VisualStudioCodeClient_Welcome Screen_Screen Reader:Screen reader is not announcing the name of the link.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#141860]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCode-Win32-Nov2021; #Win32; #ScreenReader; #Narrator; #NVDA; WCAG1.3.1; #FTP; #STP; #AINotInScope;

### Environment Details:
Application Name: Visual Studio Code Client
Windows Version: Windows 11

### Repro Steps:
1)Turn on Narrator/NVDA.
2)Open Visual studio code editor
3)Using "Alt+H" key navigate to the "Help" menu bar option .
4)Using "Down" arrow key navigate to the "Welcome" item.
5) Make sure you have the following  setting in settings.json: "workbench.startupEditor": "gettingStarted"
6) Using "Alt+H" navigate to "Help" menu bar option and use "down" arrow key to navigate to "Getting started". 
7)Tab till Get start with VS code and press enter.
8)Verify whether the screen reader is announcing the complete information about the link or not.

### Actual Result:
Screen reader is not announcing the complete information about the link or image.

### Expected Result:
Screen reader should announce the complete information about the link or image.

### User Impact:
Screen reader users will face difficulty as the complete information about the link or image is not announced by the screen reader.

![A11y_VisualStudioCodeClient_Welcome Screen_Screen ReaderScreen reader is not announcing the name of the link](https://user-images.githubusercontent.com/98530069/151760948-88eecbf6-77d9-418f-a96f-0590b678ed0c.gif)

**Comment by JacksonKearl** [comment]
The "name" of the link here is actually the stated `command:...` link.
